config/bidderinfo.go

```diff
@@ -356,6 +358,44 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) > 0 {
+		if parentBidder, ok := infos[bidder.AliasOf]; ok {
+			if parentBidder.Capabilities == nil && bidder.Capabilities != nil {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			}
+			if (parentBidder.Capabilities.Site == nil && bidder.Capabilities.Site != nil) || (parentBidder.Capabilities.Site != nil && bidder.Capabilities.Site == nil) {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
```

Comment:
how about  including site capabilities in error message? makes more clearer what capabilities is not subset

```
return fmt.Errorf("site capabilities of alias should be match or subset of parent bidder")
```